src/Components/CreateImageWizard/steps/Snapshot/components/TemplatesEmpty.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import {
+  EmptyState,
+  EmptyStateVariant,
+  EmptyStateHeader,
+  EmptyStateBody,
+  EmptyStateFooter,
+  Button,
+} from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+
+import { TEMPLATES_URL } from '../../../../../constants';
+
+export default function TemplatesEmpty() {
+  const GoToTemplatesButton = () => {
+    return (
+      <Button
+        component="a"
+        target="_blank"
+        variant="link"
+        icon={<ExternalLinkAltIcon />}
+        href={TEMPLATES_URL}
+      >
+        Go to content templates
+      </Button>
+    );
+  };
+
+  return (
+    <EmptyState variant={EmptyStateVariant.lg} data-testid="empty-state">
+      <EmptyStateHeader titleText={'No Content Templates'} headingLevel="h4" />
```

Comment:
```suggestion
      <EmptyStateHeader titleText={'No content templates'} headingLevel="h4" />
```

we tend to stick to sentence case for titles and headings also.

Reply:
fixed this here and in the Empty component for repos too :)